modules/monitoring-disabling-the-local-alertmanager.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_the_local_alertmanager_{context}"]
+= Disabling the local Alertmanager
+
+A default installation of the {product-title} monitoring stack includes in the `openshift-monitoring` project an Alertmanager instance that routes alerts from Prometheus instances.
+
+You can disable this Alertmanager instance by configuring the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+You might do so in single-node deployments that do not need to deploy a local Alertmanager cluster because alerts are centralized at the core (for example, in the hub cluster) or because running Alertmanager locally would take resources away from user workloads and add management overhead.
```

Comment:
How might we improve the readability of this phrase?

Reply:
@rolfedh Which phrase at which line number?

Reply:
@bburt-rh Line 11.

Reply:
Reworked and simplified the phrasing. Thanks!